cmd/bnsd/x/account/codec.proto

```diff
@@ -31,6 +31,11 @@ message Domain {
   // Account review defines the duration of the account renewal period for each
   // account that belongs to this domain.
   int64 account_renew = 7 [(gogoproto.casttype) = "github.com/iov-one/weave.UnixDuration"];
+  // Third party token is an arbitrary data that can be provided by a middleman
+  // (i.e. retailer) that although does not participate directly in the
+  // transaction, helped a user to register a domain. Storing a third party
+  // token helps identify contribution of such party later.
+  bytes third_party_token = 8;
```

Comment:
@orkunkl, this needs to be an IOV address, not arbitrary data.  We want to use this field for automated reward distribution.  Letting it be an e-mail address or anything other than an IOV address will break our automation ambitions.  

Please rename it to `broker`.  If you have an issue with renaming then let's talk about it offline.